### PR TITLE
Add a new BBL Check-Standby command, for alerting

### PR DESCRIPTION
**Pull request by pgilmorepf**
NOTE: Code in public repo!

Reviewers:
You're mostly looking at the new file: CheckBblStandby.cs

That verifies that the regions look good for the version numbers that are important.
It takes a lot of configurable inputs:
PF_TEST_TITLE_DATA_JSON = path
BBL_VERSIONS = semicolon-separated-strings
GAP_MAX_THRESHOLDS = 6-semicolon-separated-ints
STANDBY_MIN_THRESHOLDS = 6-semicolon-separated-ints
MAX_CAPACITY_PERCENT_THRESHOLDS = 6-semicolon-separated-ints
CYCLE_DURATION_MINS = float
CYCLE_PERIOD_SEC = float
RETRY_DELAY_SEC = float
RETRY_COUNT = int

Example Output:

Found Version: 2.0.1,  BuildId: 73...f6
    - AustraliaSoutheast, (sby:480/max:30000)(sby:480/prp:0/act:24/max:30000)
    - AustraliaEast, (sby:480/max:30000)(sby:480/prp:0/act:69/max:30000)
    - WestUs, (sby:480/max:23000)(sby:293/prp:12/act:2670/max:23000)
SEV_5 2.0.1, 73...f6 - High "StandBy + Propping" Gap (175) in region:WestUs
    - BrazilSouth, (sby:480/max:30000)(sby:432/prp:24/act:509/max:30000)
    - CentralUs, (sby:480/max:30000)(sby:424/prp:12/act:416/max:30000)
    - EastAsia, (sby:480/max:30000)(sby:484/prp:0/act:14/max:30000)
    - EastUs, (sby:480/max:30000)(sby:224/prp:36/act:3487/max:30000)
SEV_4 2.0.1, 73...f6 - High "StandBy + Propping" Gap (220) in region:EastUs
    - EastUs2, (sby:480/max:30000)(sby:401/prp:0/act:958/max:30000)
    - JapanEast, (sby:480/max:30000)(sby:476/prp:0/act:23/max:30000)
    - JapanWest, (sby:480/max:30000)(sby:480/prp:0/act:9/max:30000)
    - NorthCentralUs, (sby:480/max:30000)(sby:301/prp:60/act:2330/max:30000)
SEV_5 2.0.1, 73...f6 - High "StandBy + Propping" Gap (119) in region:NorthCentralUs
    - NorthEurope, (sby:480/max:30000)(sby:471/prp:0/act:2775/max:30000)
    - SouthCentralUs, (sby:480/max:23000)(sby:186/prp:48/act:3441/max:23000)
SEV_4 2.0.1, 73...f6 - High "StandBy + Propping" Gap (246) in region:SouthCentralUs
    - SoutheastAsia, (sby:480/max:30000)(sby:482/prp:0/act:18/max:30000)
    - WestEurope, (sby:480/max:25000)(sby:561/prp:0/act:2857/max:25000)

Found Version: 3.0.0,  BuildId: 1a...fc
    - EastUs, (sby:36/max:1000)(sby:36/prp:0/act:0/max:1000)
    - AustraliaEast, (sby:12/max:3000)(sby:12/prp:0/act:0/max:3000)
    - AustraliaSoutheast, (sby:12/max:1500)(sby:12/prp:0/act:0/max:1500)
    - CentralUs, (sby:12/max:1500)(sby:12/prp:0/act:0/max:1500)
    - EastAsia, (sby:12/max:1000)(sby:12/prp:0/act:0/max:1000)
    - EastUs2, (sby:36/max:1000)(sby:36/prp:0/act:0/max:1000)
    - JapanEast, (sby:12/max:1500)(sby:12/prp:0/act:0/max:1500)
    - JapanWest, (sby:12/max:1500)(sby:12/prp:0/act:0/max:1500)
    - NorthEurope, (sby:36/max:3600)(sby:36/prp:0/act:0/max:3600)
    - SoutheastAsia, (sby:12/max:1500)(sby:12/prp:0/act:0/max:1500)
    - WestEurope, (sby:12/max:9600)(sby:12/prp:0/act:0/max:9600)
    - WestUs, (sby:36/max:10800)(sby:47/prp:0/act:1/max:10800)
    - NorthCentralUs, (sby:12/max:4200)(sby:12/prp:0/act:0/max:4200)
    - SouthCentralUs, (sby:12/max:10200)(sby:26/prp:0/act:22/max:10200)
    - BrazilSouth, (sby:12/max:1800)(sby:12/prp:0/act:0/max:1800)

Thunderhead alert list:

2.0.1, 73...f6 - High "StandBy + Propping" Gap (220) in region:EastUs
2.0.1, 73...f6 - High "StandBy + Propping" Gap (246) in region:SouthCentralUs

Finished Query Cycle
NONE  <- Worst Bumblelion Error Level
SEV_4  <- Worst Thunderhead Error Level
NONE  <- Typical Error Level. Total Number of Alerts: 11
Done! Press any key to close

### CODE REVIEW

File: JenkinsConsoleUtility/jcuSrc/Commands/CheckBblStandby.cs [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+using PlayFab.MultiplayerModels;
+using System.Linq;
+
+namespace JenkinsConsoleUtility.Commands
+{
+    public class CheckBblStandby : ICommand
+    {
+        private enum Severity : byte
+        {
+            SEV_0, // Wake up everyone
```
Comment (Toddsa) [inline JenkinsConsoleUtility/jcuSrc/Commands/CheckBblStandby.cs]: Does not exist

File: JenkinsConsoleUtility/jcuSrc/Commands/VersionVarWriter.cs [hunk 2]
```diff
@@ -41,16 +42,16 @@ public int Execute(Dictionary<string, string> argsLc, Dictionary<string, string>
                 _apiSpecPfUrl = JenkinsConsoleUtility.GetArgVar(argsLc, "apiSpecPfUrl");
             else
             {
-                JenkinsConsoleUtility.FancyWriteToConsole("Api-Spec input not defined.  Please input one of: apiSpecPath, apiSpecGitUrl, apiSpecPfUrl");
+                JcuUtil.FancyWriteToConsole("Api-Spec input not defined.  Please input one of: apiSpecPath, apiSpecGitUrl, apiSpecPfUrl");
```
Comment (toddbellMSFT) [inline JenkinsConsoleUtility/jcuSrc/Commands/VersionVarWriter.cs]: nit: I think I have seen JcuUtil in a few other places, but wouldn't it properly shorten to JcUtil?

File: JenkinsConsoleUtility/jcuSrc/Util/Util.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections;
+
+namespace JenkinsConsoleUtility.Util
+{
+    public static class JcuUtil
```
Comment (toddbellMSFT) [inline JenkinsConsoleUtility/jcuSrc/Util/Util.cs]: Well I guess it makes sense with this namespace being jcu.Util

File: JenkinsConsoleUtility/jcuSrc/Commands/VersionVarWriter.cs (hunk 2 above)
Comment (pgilmorepf) [reply JenkinsConsoleUtility/jcuSrc/Commands/VersionVarWriter.cs]: Lol
You don't like Jenkins Console Utility Util?

**Pull request merged**